pkg/antctl/raw/set/flow-aggregator/command.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flow_aggregator
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/flowaggregator/util"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	flowaggregatorconfig "antrea.io/antrea/pkg/config/flowaggregator"
+)
+
+// Command is the support bundle command implementation.
+var Command *cobra.Command
+
+type FlowAggregatorConfigMutator func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error
+
+var mutators map[string]FlowAggregatorConfigMutator
+
+var example = strings.Trim(`
+  Enable ClickHouse
+  $ antctl set flow-aggregator clickHouse.enable=true
+  Update ClickHouse database
+  $ antctl set flow-aggregator clickHouse.database=name
+  Update ClickHouse databaseURL
+  $ antctl set flow-aggregator clickHouse.databaseURL=http://xxxxx
+  Update ClickHouse debug
+  $ antctl set flow-aggregator clickHouse.debug=true
+  Update ClickHouse compress
+  $ antctl set flow-aggregator clickHouse.compress=true
+  Update ClickHouse commitInterval
+  $ antctl set flow-aggregator clickHouse.commitInterval=10s
+  Update IPFIX Flow Collector address
+  $ antctl set flow-aggregator flowCollector.address=<IP>:<port>[:<proto>]
+  Enable IPFIX Flow Collector
+  $ antctl set flow-aggregator flowCollector.enable=true
+`, "\n")
+
+func NewFlowAggregatorSetCommand() *cobra.Command {
+	Command = &cobra.Command{
+		Use:     "flow-aggregator",
+		Short:   "Update ClickHouse/IPFIX Flow Collector parameters used in Flow Aggregator",
```

Comment:
```suggestion
		Short:   "Update configuration parameters used in Flow Aggregator",
```

I don't think this needs to be limited to collector parameters, even though that's what we support today

Reply:
Make sense. I've modified it. Thanks.